src/infra/postgres/client/client.go

```diff
@@ -3,15 +3,17 @@ package client
 import (
 	"context"
 
+	"github.com/go-pg/pg/v10"
 	"github.com/go-pg/pg/v10/orm"
 
+	"github.com/consensys/quorum-key-manager/pkg/errors"
 	"github.com/consensys/quorum-key-manager/src/infra/postgres"
-	"github.com/go-pg/pg/v10"
 )
 
 type PostgresClient struct {
-	cfg *Config
-	db  orm.DB
+	cfg  *Config
+	db   orm.DB
+	pgdb *pg.DB
```

Comment:
pgdb = db where db is the interface and pgdb is the struct so here you should not need pgdb

Reply:
I needed access to be able to ping the DB, `orm.DB` doesn't offer this functionality.

Reply:
Finally I removed it with the tests not using dockertest anymore.